### ISSUE TITLE
Broken Link for Training/Validation Sets & Inquiry about Test Set Release Timeline

### ISSUE THREAD

**Issue by vettelx** [manishitIITB/Visual_Metaphor_ImageMet_EMNLP2025#1]
Thank you for this repository. I have two questions regarding the data availability:

Broken Training Data Link: I noticed that the link to the Training/Validation Sets (in the sentence "You can find the dataset here") appears to be invalid or broken. Could you please update it?

Test Set Update: The documentation mentions that you will soon populate the Hugging Face space with proxy images/urls for the Test Set. Do you have an estimated timeline for when this will be available?

Looking forward to your response. Thanks!

**Comment by illusionnnnnnn** [comment]
Same question.  Training/Validation Sets (in the sentence "You can find the dataset here") appears to be invalid or broken. Could you please update it?
